tests/tasks/webhook-gitlab/git-update-lagoon-commit-push.yaml

```diff
@@ -0,0 +1,38 @@
+- name: "{{ testname }} - make sure we are on the branch {{ branch }}"
+  shell: git checkout -b {{ branch }} || git checkout {{ branch }}
+  args:
+    chdir: /{{git_repo_name}}
+
+#This is where we need to move the .lagoon.yml.test over the .lagoon.yml file
```

Comment:
let's ass the move functionaly to `git-add-commit-push.yaml` like we already do for the `docker_files` variable, we can have a functionality which will take a filename (like `.lagoon-no-cronjobs.yml`) as an argument, and if the variable is filled ansible will automatically rename it for you.

Reply:
Done in 7e4bba47